submitter/code_axs.py

```diff
@@ -627,10 +661,10 @@ def extract_map(accuracy_metric):
             status = get_result_status(mlperf_log_path)
 
         else:
-            if (target_accuracy[model_name]) <= float(actual_accuracy[model_name]):
+            if (target_accuracy[mlperf_model_name]) <= float(actual_accuracy[mlperf_model_name]):
```

Comment:
Is this where we want to use the new validate functionality (possibly multiple metrics, possibly ranges instead of single values)?